deltachat-jsonrpc/src/api/types/chat.rs

```diff
@@ -159,6 +161,8 @@ pub struct BasicChat {
     /// in the contact profile
     /// if 1:1 chat with this contact exists and is protected.
     is_protected: bool,
+
+    is_encrypted: bool,
```

Comment:
Not sure what to do about the documentation here; just duplicate it?

```suggestion
    /// True if the chat is encrypted.
    /// This means that all messages in the chat are encrypted,
    /// and all contacts in the chat are "pgp-contacts",
    /// i.e. identified by the PGP key fingerprint.
    ///
    /// False if the chat is unencrypted.
    /// This means that all messages in the chat are unencrypted,
    /// and all contacts in the chat are "email-contacts",
    /// i.e. identified by the email address.
    /// The UI should mark this chat e.g. with a mail-letter icon.
    ///
    /// Unencrypted groups are called "ad-hoc groups"
    /// and the user can't add/remove members,
    /// create a QR invite code,
    /// or set an avatar.
    /// These options should therefore be disabled in the UI.
    ///
    /// Note that it can happen that an encrypted chat
    /// contains unencrypted messages that were received in core <= v1.159.*
    /// and vice versa.
    ///
    /// See also `is_pgp_contact` on `Contact`.
    is_encrypted: bool,
```